src/Modules/OrchardCore.Commerce/Services/StripePaymentService.cs

```diff
@@ -286,6 +266,33 @@ public async Task UpdateOrderToPaymentFailedAsync(PaymentIntent paymentIntent)
         return order;
     }
 
+    public async Task<IList<OrderLineItem>> CreateOrderLineItemListAsync(ShoppingCart shoppingCart)
+    {
+        IList<OrderLineItem> lineItems = new List<OrderLineItem>();
```

Comment:
Can't this be `var`?

Reply:
You are right. I didn't really check these moved codes for these tbh.